process/charter.html

```diff
@@ -292,9 +285,9 @@ <h3>5.1 Organizing the Call for Review</h3>
     be sent <strong>at least three business days before</strong> the anticipated start
     date of the review. The request must include:
     <ol>
-      <li>A URI to the proposed charter (Note: avoid using github.io links); this charter is public during the AC review.</li>
+      <li>A URI to the proposed charter; please don't use github.io links. This charter is public during the AC review.</li>
```

Comment:
I think the underlying concern is less about github.io but that the content may change while it is still under review (including horizontal review).

Thus, focus should instead be on immutable snapshots of a proposed charter. This should either be on:

*  W3C controlled space (e.g., a W3C Team controlled GitHub repository); or
* W3C URI space (w3.org); or
* a well-known trusted third-party service (e.g., archive.org).

This not only ensures that review are based on fixed content but also would prohibit changes to the proposed charter by anyone that is either unauthorised or did not have the consent of the group that's proposing the charter, e.g., see https://github.com/solid/solid-wg-charter/pull/47#issuecomment-1683686486 where it talks about this exact issue:

>The proposed WG charter is still under horizontal review and making changes along the lines proposed in this PR without CG's consent is unacceptable behaviour.

Reply:
Team practice is to always put a copy of the proposed charter in W3C url space for ac review.

Personally I use a naming convention so the Foo WG charter will be like `foo-2023-ac.html` which also makes it easier to get review of post-ac changes by sending a diff between that and `foo-2023.html`.

But yes the point is to avoid changing the document which is under AC review.